website/docs/r/signalr_web_pubsub_hub.html.markdown

```diff
@@ -1,14 +1,14 @@
 ---
 subcategory: "Web Publishing Subscription"
 layout: "azurerm"
-page_title: "Azure Resource Manager: azurerm_web_pubsub_hub"
+page_title: "Azure Resource Manager: azurerm_signalr_web_pubsub_hub"
```

Comment:
since we're reverting the `signalr_` prefix, we should be able to revert the docs here?